docs/db2graph/db2graph.rst

```diff
@@ -0,0 +1,279 @@
+    
+       .. code-block:: bash
+    
+        $ git clone https://github.com/guenthermi/the-movie-database-import.git 
+        $ cd the-movie-database-import
+        $ vi db_config.json # update the username, password, host, and db_name fields if applicable. Here, password is changed to 'password' and db_name is 'test_db'
+        $ mkdir dataset/ # create a dataset directory and place the downloaded dataset file in it
+        $ unzip archive.zip # unzip the downloaded dataset file
+        $ python3 loader.py dataset/ # load the dataset files from the path to your dataset folder
+        $ psql -U postgres -d test_db # check what is inside the database now
+        > postgres=# \d
+                                 List of relations
+         Schema |                Name                |   Type   |  Owner
+        --------+------------------------------------+----------+----------
+         public | actors                             | table    | postgres
+         public | actors_id_seq                      | sequence | postgres
+         ...
+        (30 rows)    
+   
+   | This creates 15 tables containing information about actors, movies, keywords, production companies, production countries, as well as credits data.
+   
+   | Install Marius and the required dependencies for Db2Graph.
+   
+   .. code-block:: bash 
+       
+       $ cd / # back to root directory
+       $ apt-get update
+       $ apt-get install vim
+       $ apt-get install git
+       $ apt-get install python3
+       $ apt-get install python3-pip
+       $ pip3 install psycopg2-binary
+       $ pip3 install mysql-connector-python
+       $ pip3 install hydra-core --upgrade
+       $ git clone https://github.com/marius-team/marius.git
+       $ pip3 install. 
+
+#. | Next, create the configuration files. From the root directory, navigate to the Db2Graph directory and create the ``conf/config.yaml``, ``conf/entity_nodes.txt``, ``conf/edges_entity_entity.txt``, and ``conf/edges_entity_feature_values.txt`` files if they have not been created. 
+
+    .. code-block:: bash 
+       
+       $ cd Marius/src/python/tools/db2graph/
+       $ vi conf/config.yaml
+
+   | In ``conf/config.yaml``, define the following fields:
+    
+    .. code-block:: yaml
+        
+            db_server: postgre-sql
+            db_name: test_db
+            db_user: postgres
+            db_password: password
+            db_host: 127.0.0.1
+            generate_uuid: false 
+            entity_node_queries: conf/entity_nodes.txt
+            edges_entity_entity_queries: conf/edges_entity_entity.txt
+            edges_entity_feature_values_queries: conf/edges_entity_feature_values.txt
+
+   | In ``conf/edges_entity_entity.txt``, define the following queries. Note that we create three edges/relationships: An actor acted in a movie; A movie directed by a director; A movie produced by a production company.
```

Comment:
What about the contents of entity_nodes.txt and edges_entity_feature_values.txt?

Reply:
They are not needed in this example. Added a line saying that.

The majority of use cases users will only need to use `edges_entity_entity.txt` only without any need for the other 2 files.